.eslintrc.yml

```diff
@@ -5,6 +5,13 @@ parserOptions:
   ecmaVersion: 12
   sourceType: module
   allowImportExportEverywhere: true
+plugins:
+  - svelte3
+overrides:
+  -
```

Comment:
This line looks funny. Shouldn't there be a name beyond the hyphen? I'm not too familiar with the YAML syntax in this scenario.

Reply:
Yeah it's weird, but this creates an object which is what the plugin needed. I tried a couple of other ways of writing it, non of it would do the job.